free-2.md

```diff
@@ -11,34 +10,50 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for this scenario because it allows the user to do a specific set of tasks related to a brewing simulation. It would not require users to be technical and it would not try to do more beyond addressing this need. It has a very clear purpose and the commands would need to be very specific depending on the variables entered by the user. 
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have recently been doing research on homebrewing and am only beginning to discover all of the factors that affect the outcomes of the product. Drawing on many different sources, I attempted to craft my own recipe to achieve the end result that I want. I think that this language would be very helpful for someone like me who knows what they want from an end product but can't find a specific recipe to suit their needs. Also, one batch of brew is a significant time investment and it would be helpful to have more of an idea of the end product before putting weeks into the process.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Simulated outcomes for brewing conditions.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Since the language would be aimed at users with a wide variety of technical backgrounds, I think that a GUI (or something similar) would be optimal. The GUI could contain multiple fields or a series of questions in which users input their brewing conditions. These parameters are adjustable by the user and comprise a program. As stated before, this is correct for the domain because it will allow a wide variety of users to construct simulations.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When a program runs, the GUI would output the result of the brew. This could provide a variety of information depending on what would be brewed. It might include the sweetness/dryness of the end product and the amount of carbonation. The language could possibly simulate the brew over time and show the state of the brew (give statistics and the aforementioned information) at any time step in the process. This may involve an animation or graphs of certain variables over time. Any errors in compiling the program would be communicated in boxes prompting the user to input valid values for a certain field. A different type of error would be if the brew would explode with the user's given conditions, which could be shown in an animation or a visual message.
```

Comment:
I'm curious if you've thought ahead at all about how you might actually simulate the process. I'm sure brewing has been broken down to an exact science at this point, but I imagine it's still difficult to know exactly what will happen (otherwise we wouldn't even need this simulation). Like any simulation, I imagine to obtain even more accuracy requires an exponential amount of more work.